plugins/auth-backend/src/lib/OAuthProvider.ts

```diff
@@ -103,6 +130,7 @@ export class OAuthProvider implements AuthProviderRouteHandlers {
   async start(req: express.Request, res: express.Response): Promise<void> {
     // retrieve scopes from request
     const scope = req.query.scope?.toString() ?? '';
+    const env = req.query.env?.toString() ?? '';
```

Comment:
Same goes for this. If no `env` is set we want to fail early and throw an `InputError`

Reply:
start now throws an Error if `env` is empty or missing.